scripts/components/keepfollowing.lua

```diff
@@ -236,6 +271,163 @@ function KeepFollowing:SetLeader(leader)
     return false
 end
 
+--- Wormhole
+-- @section wormhole
+
+local function ResetWormholeFields(self)
+    -- wormhole
+    self.is_wormhole_travelling = false
+    self.is_wormhole_travel_successful = false
+    self.last_leader_pos = nil
+    self.leader_id = nil
+    self.leader_used_wormhole = false
+    self.wormhole_travel_thread = nil
+end
+
+local function LocateNearbyWormhole(pos)
+    local wormhole
+    local entities = TheSim:FindEntities(pos.x,pos.y,pos.z,2,{"teleporter"},{})
+    local wormhole_prefabs = {"wormhole","tentacle_pillar_hole","pocketwatch_portal_entrance"}
+    for _,entity in pairs(entities) do
+        for _,prefab in pairs(wormhole_prefabs) do
+            if entity.prefab == prefab then
+                wormhole = entity
+                break
+            end
+        end
+    end
+    return wormhole
+end
+
+local function JumpThroughWormhole(self,wormhole)
```

Comment:
This should become a part of SDK as part of https://github.com/dstmodders/dst-mod-sdk/issues/6. Just a note, since you've mentioned that in Slack.